Cargo.toml

```diff
@@ -9,7 +9,7 @@ categories = ["network-programming", "web-programming::http-server"]
 license = "MIT"
 build = "build.rs"
 # Remember to also update in appveyor.yml and the http-crates.io branch
-version = "2.1.0"
+version = "2.1.1"
```

Comment:
Not that version number really matters, but I feel this is more than just a patch bump :woman_shrugging:

Reply:
This was patch-sized but then it grew in the mean-time. Bumped to 2.2.0.